mono/metadata/gc.c

```diff
@@ -780,7 +781,7 @@ finalize_domain_objects (void)
 	DomainFinalizationReq *req = NULL;
 	MonoDomain *domain;
 
-	if (domains_to_finalize) {
+	if (UnlockedReadPointer ((void *)&domains_to_finalize)) {
```

Comment:
should be `gpointer` instead of `void *`

Reply:
of course; sorry forgot! (fixed)